sdk/core/azure-core/src/main/java/com/azure/core/exception/HttpResponseError.java

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.exception;
+
+import java.util.List;
+
+/**
+ * This class represents the error details of an HTTP response.
+ */
+public final class HttpResponseError {
```

Comment:
Should this class and `HttpResponseErrorInner` be annotated with Jackson properties to enable deserialization?